frontend/src/pages/Domains/index.tsx

```diff
@@ -79,26 +90,96 @@ const Domains: React.FC = () => {
         <Typography className={classes.title} align="center">
           DEVELOPERS
         </Typography>
+        <Typography className={classes.subtitle} align="left">
+          2020 Batch
+        </Typography>
+        <div className={classes.flexbox}>
+          {teamList.map((memberData, i) => {
+            return memberData.team === "dev" && memberData.batch === 2020 ? (
+              <DomainCard key={i} {...{ ...memberData, team: "Developer" }} />
+            ) : null;
+          })}
+        </div>
```

Comment:
This could be a separate component. If not a component then can be looped over each batch's data.

Reply:
Yes. We should do it in a way such that we don't have to make changes here for every batch.

As @Samy-33 mentioned, one way is to extract all the batches from the list and loop over them.